clients/python/sliderule/earthdata.py

```diff
@@ -178,6 +178,35 @@ def __cmr_granule_metadata(search_results):
     # - polygons as geodataframe geometry
     return granule_metadata
 
+def __cmr_collection_query(provider, short_name):
+    """Perform a CMR query for collection metadata."""
+    # build params
+    cmr_query_type = 'collections'
+    cmr_format = 'json'
+    CMR_URL = 'https://cmr.earthdata.nasa.gov'
+    cmr_query_url = '/'.join([CMR_URL, 'search', f'{cmr_query_type}.{cmr_format}'])
+    # build list of CMR query parameters
+    params = []
+    params.append(f'?provider={provider}')
+    params.append(f'&short_name={short_name}')
+    # full CMR query url
+    cmr_query_url += "".join(params)
+    logger.debug(f'cmr request={cmr_query_url}\n')
+    # ssl context
+    ctx = ssl.create_default_context()
+    ctx.check_hostname = False
+    ctx.verify_mode = ssl.CERT_NONE
```

Comment:
out of curiosity what does this do is it necessary?

Reply:
sometimes urllib throws ssl errors if a context is not provided

Reply:
I believe this disables cert verification.  So it protects against man-in-the-middle and things like that.  There are a handful of reasons a valid cert could fail to be verified; and while I know technically we shouldn't do this, it appears in the code for CMR access elsewhere and I don't think it is overly risky as there is no sensitive information being passed in either direction.